notification/index.lambda.js

```diff
@@ -1,10 +1,11 @@
-const handler = require('./handler/notification/notification.handler')
-const logger = require('./utils/logger').getLogger()
-const { getNotificationForTracking } = require('./utils/commons')
-const { getCtpProjectConfig, getAdyenConfig } = require('./utils/parser')
+const handler = require('./src/handler/notification/notification.handler')
+const logger = require('./src/utils/logger').getLogger()
+const { getNotificationForTracking } = require('./src/utils/commons')
+const { getCtpProjectConfig, getAdyenConfig } = require('./src/utils/parser')
 
 exports.handler = async (event) => {
-  const { notificationItems } = event
+  const body = event.body ? JSON.parse(event.body) : event
```

Comment:
Shall we update corresponding unit test case ? so that the mocking event in unit test contains body attribute.

Reply:
> there is a body attribute always inside event object. Shall we update corresponding unit test case as well ?
We were not sure about it and thought it could be related to the API gateway, can you ensure it ? If yes we could update it, for now we are not sure.

Reply:
here: https://stackoverflow.com/a/41656022

Reply:
That means we have two options when establishing API gateway on lambda. I think it would be fine to keep unit test unchanged at this moment which pretends request from lambda integration, since we cannot 100% sure the json format for lambda proxy integration until we adapt it.

Reply:
The check seems obsolete/not obvious. Why we do not add to such places some comments or links?